Fixes #2333 Adds Russian translations

dega1989 opened the pull request:
Fixes #2333

Making it more native for Russian speaking persons!

dega1989 committed:
Fixes #2333 Adds Russian translations

Fixes #2333

Making it more native for Russian speaking persons!

annadanchenko reviewed (changes requested):
"Извените" is not correct. Please, re-check spelling and grammar

rasom commented on src/status_im/translations/ru.cljs:
```diff
@@ -2,308 +2,363 @@
+   :edit-network-warning                  "Бкдьте осторожны, изменение данных сети может отключить вас от нее"
+   :connecting-requires-login             "Для подключение к другой сети необходимо авторизироваться"
+   :close-app-title                       "Острожно!"
+   :close-app-content                     "Приложение будет остановленно и закрыто. Когда вы его снова откроете, булкт использована выбранная сеть"
```
`булкт`

dega1989 committed:
Update ru.cljs

dega1989 reviewed (commented):
Changed "Извените" for "Извините".
str №114

annadanchenko commented:
@dega1989 please re-check spelling and grammar for all of your translations. There are too many typos, I can't list them all

dega1989 committed:
Update ru.cljs

Str №18, 87, 114, 155, 158, 174, 209, 271, 275, 280, 284, 284, 290, 300, 319, 325, 326, 327, 360, 361, 363

dega1989 commented:
Update ru.cljs
Str №18, 87, 114, 155, 158, 174, 209, 271, 275, 280, 284, 284, 290, 300, 319, 325, 326, 327, 360, 361, 363.

annadanchenko commented:
Typos still survived after the spell check. For example "Количестро", "Сслылка". I didn't run the full check as it looks like what this PR is about - translate it in a nice way, so Russian-speaking audience can enjoy the app in Russian.

dega1989 committed:
Update ru.cljs

Str 26. Is this translation for Android version? 
Alternatives
48  :faucet-success                            "Был получен запрос на открытие крана"
49  :faucet-error                                "Ошибка запроса на открытие крана"
105:browsing-browse                        "@бзор"

dega1989 reviewed (commented):
Check This one!

rasom commented:
closing this PR because https://github.com/status-im/status-react/pull/2361 was already merged
